src/main/java/net/mcreator/ui/dialogs/NewModElementDialog.java

```diff
@@ -42,6 +43,8 @@ public static void showNameDialog(MCreator mcreator, ModElementType<?> type) {
 		regName.setForeground(Theme.current().getAltForegroundColor());
 		regName.setBorder(BorderFactory.createEmptyBorder(5, 0, 5, 0));
 
+		WorkspaceFolderBreadcrumb breadcrumb = WorkspaceFolderBreadcrumb.createDialogBreadcrumb(mcreator);

```

Comment:
I asked for an object with a method like "makeScrollable" or something like this. Instead I got two static methods.

Those two methods can be used outside designated pattern so this is not ok. Make object that defines this more clearly.

This is also better called small breadcrumb, not dialog breadcrumb, as use of it is not limited to dialogs in my mind